pkg/parser/parser.peg

```diff
@@ -363,6 +363,81 @@ StandaloneAttributes <- attributes:(ElementAttribute)+ BlankLine* EOF { // stand
     return types.NewAttributes(attributes)
 }
 
+// ------------------------------------------
+// Quoted Strings (between curly single or double quotes)
+// ------------------------------------------
+
+QuotedString <- qs:(SingleQuotedString / DoubleQuotedString) {
+    return qs, nil
+}
+
+SingleQuotedString <- "'`" elements:SingleQuotedStringElements "`'" {
+    return types.QuotedString{ Kind: types.SingleQuote, Elements: elements.([]interface{})}, nil
+}
+
+SingleQuotedStringElements <- elements:(SingleQuotedStringElement+) {
+    return types.NewInlineElements(elements)
+}
+
+// We have to treat this one special, because of ambiguity with monospace markup.
+SingleQuotedStringElement <- element:(
+        LineBreak // must be before spaces
+        / Space+
+        / InlineIcon
+        / InlineImage
+        / InlineFootnote
+        / InlinePassthrough
+        / Link
+        / AttributeSubstitution
+        / BoldText
+        / ItalicText
+        / MarkedText
+        / SubscriptText
+        / SuperscriptText
+        / !"`'" MonospaceText
+        / DoubleQuotedString
+        / SingleQuotedStringFallbackCharacter) {
+    return element, nil
+}
+
+SingleQuotedStringFallbackCharacter <-  [^\r\n`] / "`" !"'" {
+    return types.NewStringElement(string(c.text))
+}
+
+DoubleQuotedString <- "\"`" elements:DoubleQuotedStringElements "`\"" {
+
+    return types.QuotedString{ Kind: types.DoubleQuote, Elements: elements.([]interface{})}, nil
```

Comment:
same here, let's add and use `types.NewQuotedString(...)`

Reply:
Done.